src/styles.scss

```diff
@@ -0,0 +1,2 @@
+@tailwind base;
+@tailwind utilities;
```

Comment:
I do not see tailwind config file. Explore themes as well

Reply:
Done sir